### DISCUSSION TITLE
can I use middle mouse to move and click to autoattack?

### DISCUSSION THREAD

**Discussion by marcspc** [hwnd-git/Diablo-IV-AHK-WASD#5] [General]
feels weird not being able to click for attacking, tried editing
	ControlClick, x%xCenter% y%yCenter%, A,, M, 1, NA

on the 3 repetitions and remap movement to middle click, but then it moves toward real mouse position when pressing any movement key (wasd)

**Comment by hwnd-git** [comment]
Unfortunately, I experience the exact same behavior and this is the main reason why I went for completely abandoning LMB for action. I have main skill binded to Spacebar and secondary to LCtrl, interaction is F. I have read some YouTube comments though, saying that MMB for movement works for some people. But it's a mystery for my as to why. If I had to guess, maybe it's tied to mouse drivers... Do you happen to have a gaming mice with programmable buttons (I do)?

**Reply by marcspc** [reply]
yes, razer naga v2 pro
